contracts/minitoken/solidity/migrations/2_token_migration.js

```diff
@@ -35,27 +29,26 @@ const deployCore = async (deployer) => {
   await deployer.deploy(IBCChannel);
   await deployer.link(IBCChannel, [IBCHandler]);
 
-  await deployer.deploy(MockClient);
+  await deployer.deploy(IBCChannelHandshake);
+  await deployer.deploy(IBCPacket);
+  await deployer.deploy(IBCHandler, IBCClient.address, IBCConnection.address, IBCChannelHandshake.address, IBCPacket.address);
```

Comment:
You can make these lines simpler like below.
https://github.com/hyperledger-labs/yui-ibc-solidity/blob/main/migrations/2_deploy_contracts.js#L14-L18

Reply:
https://github.com/hyperledger-labs/yui-docs/pull/67/commits/764ab05bd418839f936ad101db40511fd67447ba
Refactoring migrations